src/Business Foundation/App/NoSeries/src/Single/NoSeriesImpl.Codeunit.al

```diff
@@ -429,6 +429,13 @@ codeunit 304 "No. Series - Impl."
         exit(OriginalNoSeriesCode);
     end;
 
+    procedure IsNoSeriesInDateOrder(NoSeriesCode: Code[20]): Boolean
+    var
+        NoSeries: Record "No. Series";
+    begin
+        exit(NoSeries.Get(NoSeriesCode) and NoSeries."Date Order")
```

Comment:
I like the more explicit code of IsAutomatic for example. The result is the same but the code is easier to read...